pkg/apis/direct.csi.min.io/v1beta3/conversion_generated.go

```diff
@@ -0,0 +1,407 @@
+	}
+	return nil
+}
+
+// Convert_v1beta2_DirectCSIDriveList_To_v1beta3_DirectCSIDriveList is an autogenerated conversion function.
+func Convert_v1beta2_DirectCSIDriveList_To_v1beta3_DirectCSIDriveList(in *v1beta2.DirectCSIDriveList, out *DirectCSIDriveList, s conversion.Scope) error {
+	return autoConvert_v1beta2_DirectCSIDriveList_To_v1beta3_DirectCSIDriveList(in, out, s)
+}
+
+func autoConvert_v1beta3_DirectCSIDriveSpec_To_v1beta2_DirectCSIDriveSpec(in *DirectCSIDriveSpec, out *v1beta2.DirectCSIDriveSpec, s conversion.Scope) error {
+	out.RequestedFormat = (*v1beta2.RequestedFormat)(unsafe.Pointer(in.RequestedFormat))
+	out.DirectCSIOwned = in.DirectCSIOwned
+	out.DriveTaint = *(*map[string]string)(unsafe.Pointer(&in.DriveTaint))
+	return nil
+}
+
+// Convert_v1beta3_DirectCSIDriveSpec_To_v1beta2_DirectCSIDriveSpec is an autogenerated conversion function.
+func Convert_v1beta3_DirectCSIDriveSpec_To_v1beta2_DirectCSIDriveSpec(in *DirectCSIDriveSpec, out *v1beta2.DirectCSIDriveSpec, s conversion.Scope) error {
+	return autoConvert_v1beta3_DirectCSIDriveSpec_To_v1beta2_DirectCSIDriveSpec(in, out, s)
+}
+
+func autoConvert_v1beta2_DirectCSIDriveSpec_To_v1beta3_DirectCSIDriveSpec(in *v1beta2.DirectCSIDriveSpec, out *DirectCSIDriveSpec, s conversion.Scope) error {
+	out.RequestedFormat = (*RequestedFormat)(unsafe.Pointer(in.RequestedFormat))
+	out.DirectCSIOwned = in.DirectCSIOwned
+	out.DriveTaint = *(*map[string]string)(unsafe.Pointer(&in.DriveTaint))
+	return nil
+}
+
+// Convert_v1beta2_DirectCSIDriveSpec_To_v1beta3_DirectCSIDriveSpec is an autogenerated conversion function.
+func Convert_v1beta2_DirectCSIDriveSpec_To_v1beta3_DirectCSIDriveSpec(in *v1beta2.DirectCSIDriveSpec, out *DirectCSIDriveSpec, s conversion.Scope) error {
+	return autoConvert_v1beta2_DirectCSIDriveSpec_To_v1beta3_DirectCSIDriveSpec(in, out, s)
+}
+
+func autoConvert_v1beta3_DirectCSIDriveStatus_To_v1beta2_DirectCSIDriveStatus(in *DirectCSIDriveStatus, out *v1beta2.DirectCSIDriveStatus, s conversion.Scope) error {
+	out.Path = in.Path
+	out.AllocatedCapacity = in.AllocatedCapacity
+	out.FreeCapacity = in.FreeCapacity
+	out.RootPartition = in.RootPartition
+	out.PartitionNum = in.PartitionNum
+	out.Filesystem = in.Filesystem
+	out.Mountpoint = in.Mountpoint
+	out.MountOptions = *(*[]string)(unsafe.Pointer(&in.MountOptions))
+	out.NodeName = in.NodeName
+	out.DriveStatus = v1beta2.DriveStatus(in.DriveStatus)
+	out.ModelNumber = in.ModelNumber
+	out.SerialNumber = in.SerialNumber
+	out.TotalCapacity = in.TotalCapacity
+	out.PhysicalBlockSize = in.PhysicalBlockSize
+	out.LogicalBlockSize = in.LogicalBlockSize
+	out.Topology = *(*map[string]string)(unsafe.Pointer(&in.Topology))
+	out.AccessTier = v1beta2.AccessTier(in.AccessTier)
+	// INFO: in.FilesystemUUID opted out of conversion generation
+	// INFO: in.PartitionUUID opted out of conversion generation
+	// INFO: in.MajorNumber opted out of conversion generation
+	// INFO: in.MinorNumber opted out of conversion generation
```

Comment:
Looks like these fields are opted out for conversion, We need to remove the `+k8s:conversion-gen=false` comment on these fields in `pkg/apis/direct.csi.min.io/v1beta3/types.go`

Reply:
Done